### ISSUE TITLE
Floating tooltips incorrectly positioned

### ISSUE THREAD

**Issue by lemoness** [HabitRPG/habitica#9456]
We've gotten multiple reports that sometimes tooltips from the navigation bar randomly appear elsewhere on the page and persist across Habitica pages until a browser refresh. One user reports that they've noticed this behavior on Firefox with the Gems and Gold tooltips in particular.

**Comment by lemoness** [comment]
We're investigating to see if some of our recent staging deploys have fixed this.

**Comment by mandyzhou** [comment]
In case additional information is helpful (I was the user who reported noticing this behavior on Firefox with the Gems and Gold tooltips), I've since also noticed this with the Sync and User tooltips, as well as with tooltips from the Skills action drawer. Another user in the Report a Bug guild also reported seeing this behavior with Skills tooltips.

**Comment by joedragons** [comment]
If this is what you mean I noticed and took a screen, in case it helps.
Interestingly, I had not cast "Searing Brightness", so don't think it's related to activity.
I this occuring with "Sync" on my Party screen (no screenshot).
I also primarily use (and only noticed so far) on FF [Windows 10].
These don't move when you scroll, seem to be stuck in the place they appear.  I believe they disappear on refresh.

![habiticarevampfeedback18](https://user-images.githubusercontent.com/9559205/32643272-473782ac-c5a8-11e7-8c73-58d5df2976cf.png)

**Comment by majestic-owl448** [comment]
![habitica](https://user-images.githubusercontent.com/26656284/32646058-edf9bf28-c5ea-11e7-8091-95d7f56f8fe3.png)
 I have this one that stick to the line break between the chatbox and the messages below (or the same height if it's not a chat page), it appears almost every time that I click on the User Icon, stay there even changing page, and go away only on reload. Google Chrome 62.0.3202.89, on Windows 10.

Edited to add: This one moves when scrolling, it really seems stuck to that point of the page.

**Comment by joedragons** [comment]
>  This one moves when scrolling, it really seems stuck to that point of the page.

This is what I meant by "these don't move when you scroll"=)  I think this is a better phrasing.  Or alternative phrasing=)  Either way, I think we agree=)

**Comment by joedragons** [comment]
This occurred to me again today, this time in Chrome (Windows 10).  I wasn't sure if "We're investigating to see if some of our recent staging deploys have fixed this." meant on some staging site or if the fixes were staged (on the live site) so thought I'd report back.

**Comment by paglias** [comment]
@joedragons thanks for the report! We're investigating this on a private testing site that contains several changes and probably fixes the problem but it will take a few days before it can land on Habitica.com

I'll post here when it's live

**Comment by paglias** [comment]
Is this fixed now?

**Comment by joedragons** [comment]
@paglias FYI, I never had a reproducable scenario but I'll be on the lookout.

**Comment by paglias** [comment]
I'm closing this but if you notice it again leave a comment here!

**Comment by joedragons** [comment]
Wonder if this is the same issue.  Saw it today checking off a to-do (the todo had 2 parts but I only checked the first one in case that detail helps).  Hadn't seen anything else like this for a bit.
![habiticafeedback](https://user-images.githubusercontent.com/9559205/33792125-4da1eb76-dc66-11e7-9373-0d9a8f6ee7a4.png)
Windows 7, Firefox.
PS: Rereading the above, I noted report that when you scrolled it didn't move, so I tried it and it repositioned but then didn't.  I was preparing to take a screenshot and it vanished.

UPDATE: I noticed this again on MacOSX, Firefox.  It did not persist when I changed to Party (from Tasks) or returned.
